HIP/hip-869.md

```diff
@@ -58,63 +58,48 @@ Adopting a two-phase strategy, this approach facilitated the earlier release of
 
 **Personas**
 
-**Council Member** - A member of the council authorized to submit address book changes
+**Council** -  Hedera Council 
 
-**Node Operator** - Technical administrator of Hedera Consensus Nodes
+**Node Operator** - Administrator of Hedera Consensus Nodes
 
-**Add Node to Address Book**
 
-***User Story:*** As a council member, I want the ability to submit signed HAPI transactions to add a new consensus node to the Hedera network upon the next maintenance window, so that I can perform operations independently.
+***User Stories:*** 
+1. The Council submits signed HAPI transactions to add a new consensus node to the Hedera network upon the next maintenance window, for automated management of Hedera's Address book.
```

Comment:
Strictly speaking, this does not qualify as a user story IMO. A user story should answer who wants to do what and for what reason. It should focus on the problem, not the solution. The standard structure helps (but is not required).

Reply:
I'll change back.